api/v1alpha1/warehouse_types.go

```diff
@@ -566,6 +597,12 @@ type DiscoveredArtifacts struct {
 	//
 	// +optional
 	Charts []ChartDiscoveryResult `json:"charts,omitempty" protobuf:"bytes,3,rep,name=charts"`
+	// OtherResults holds the artifact references discovered by the Warehouse
+	// for all subscriptions that are not to a Git, container image, or Helm
+	// chart repository.
+	//
+	// +optional
+	OtherResults []GenericDiscoveryResult `json:"otherResults,omitempty" protobuf:"bytes,5,rep,name=otherResults"`
```

Comment:
Nit: rename to just `Other`/`Generic` instead of using the `Results` name to align more clearly with the other field names